lighthouse-core/gather/gather-runner.js

```diff
@@ -158,7 +159,9 @@ class GatherRunner {
    * @return {!Promise}
    */
   static beforePass(options, gathererResults) {
-    const pass = GatherRunner.loadBlank(options.driver);
+    const blankPage = options.config.blankPage;
```

Comment:
Yea, not sure. `blankStartPageUrl`? haha. We should stick with the existing nomenclature though. I wasn't aware of that part.